### ISSUE TITLE
Deprecated features used

### ISSUE THREAD

**Issue by runningman84** [sensu/sensu-chef#462]
I just got these warnings:
```
       Deprecated features used!
         Cloning resource attributes for rabbitmq_user[sensu] from prior resource (CHEF-3694)
       Previous rabbitmq_user[sensu]: /tmp/kitchen/cache/cookbooks/sensu/definitions/rabbitmq_credentials.rb:6:in `block in from_file'
       Current  rabbitmq_user[sensu]: /tmp/kitchen/cache/cookbooks/sensu/definitions/rabbitmq_credentials.rb:11:in `block in from_file' at 1 location:
           - /tmp/kitchen/cache/cookbooks/sensu/definitions/rabbitmq_credentials.rb:11:in `block in from_file'
         node.set is deprecated and will be removed in Chef 14, please use node.default/node.override (or node.normal only if you really need persistence) at 1 location:
           - /tmp/kitchen/cache/cookbooks/monitor/recipes/_handler_deregister.rb:30:in `from_file'
         Cloning resource attributes for directory[/etc/sensu/conf.d/filters] from prior resource (CHEF-3694)
       Previous directory[/etc/sensu/conf.d/filters]: /tmp/kitchen/cache/cookbooks/sensu/providers/json_file.rb:7:in `block in class_from_file'
       Current  directory[/etc/sensu/conf.d/filters]: /tmp/kitchen/cache/cookbooks/sensu/providers/json_file.rb:7:in `block in class_from_file' at 1 location:
```

**Comment by julian7** [comment]
The first one (in rabbitmq_credentials.rb) is straightforward. There are two ways CHEF-3694 can happen. However, the second one seems like coming from your "monitoring" cookbook, and not from sensu cookbook.

I don't have problems with sensu_json_file either. Can you show us an example code when it occurs?

**Comment by cwjohnston** [comment]
Closed by #466
